src/routes/safe/components/Balances/SendModal/screens/SendFunds/index.tsx

```diff
@@ -225,10 +226,10 @@ const SendFunds = ({
           }
 
           const setMaxAllowedAmount = () => {
-            const isSpendingLimit = tokenSpendingLimit && txType === 'spendingLimit'
+            const isSpendingLimitTx = tokenSpendingLimit && isSpendingLimit(txType)
```

Comment:
You already have this on line 151.

Reply:
Can you not reuse the other here? Or do the values change?

Reply:
I don't think I can reuse it, one is calculate inside the submit callback and the other comes from the form the form values to cap the "max spending value"

Reply:
Have you tried it?